applications/portfolioDrafts/portfolio/post.ts

```diff
@@ -0,0 +1,55 @@
+import { UpdateCommand } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { PortfolioStep } from ".././models/PortfolioStep";
+import { dynamodbClient } from ".././utils/dynamodb";
+
+const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+const CLIENT = dynamodbClient;
```

Comment:
Follow the way this was done in #40 with the `import`

Reply:
I've added code suggestions to assist with this change.